report/renderer/performance-category-renderer.js

```diff
@@ -205,17 +205,19 @@ export class PerformanceCategoryRenderer extends CategoryRenderer {
         metricsBoxesEl.appendChild(this._renderMetric(item));
       });
 
-      const descriptionEl = this.dom.find('.lh-category-header__description', element);
-      const estValuesEl = this.dom.createChildOf(descriptionEl, 'div', 'lh-metrics__disclaimer');
-      const disclaimerEl = this.dom.convertMarkdownLinkSnippets(strings.varianceDisclaimer);
-      estValuesEl.appendChild(disclaimerEl);
-
-      // Add link to score calculator.
-      const calculatorLink = this.dom.createChildOf(estValuesEl, 'a', 'lh-calclink');
-      calculatorLink.target = '_blank';
-      calculatorLink.textContent = strings.calculatorLink;
-      this.dom.safelySetHref(calculatorLink, this._getScoringCalculatorHref(category.auditRefs));
-
+      // Only add the disclaimer with the score calculator link if the category was rendered with a score gauge.
+      if (element.querySelector('.lh-gauge__wrapper')) {
```

Comment:
I used this as the condition because the disclaimer + calculator seem to be tied to the category gauge. It's easy to make this condition depend on gather mode if that would make more sense.